doc/mgr/index.rst

```diff
@@ -29,6 +29,7 @@ sensible.
     Writing modules <modules>
     Writing orchestrator plugins <orchestrator_modules>
     Dashboard module <dashboard>
+    Dashboard API <dashboard_api>
```

Comment:
Should we say "Ceph REST API" here?

Reply:
That's entirely up to the community; I can make that change if we want, though there is, of course, the separate 'REST API module that's named similarly.